js/controller.js

```diff
@@ -1472,6 +1474,25 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.publish(OO.EVENTS.SAVE_PLAYER_SETTINGS, this.state.persistentSettings);
     },
 
+    trySetAnamorphicFixState: function(enabled) {
```

Comment:
Should we start some JS docs for this file? I know the standard was not to include them, but I feel it will prove helpful for maintenance.